lib/chef/resource/windows_certificate.rb

```diff
@@ -134,9 +150,9 @@ def fetch_cert
           store.get(new_resource.source)
         end
 
-        def verify_cert
+        def verify_cert(thumbprint = new_resource.source)
```

Comment:
can you add a comment to this method while you're here

Reply:
Added comments. Thanks